proposed/runtime-binding.md

```diff
@@ -229,37 +225,80 @@ Can be specified via the following ways:
 * Environment variable: `DOTNET_ROLL_FORWARD`
```

Comment:
As mentioned below, `runtimeconfig` should probably support both global and per-framework setting.